test/FunctionalTests/Microsoft.OData.Edm.Tests/Validation/EdmErrorTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmErrorTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using Microsoft.OData.Edm.Validation;
+using Xunit;
+using Microsoft.OData.Edm.Csdl;
+
+namespace Microsoft.OData.Edm.Tests.Validation
+{
+    public class EdmErrorTests
+    {
+        private EdmLocation errorLocation;

```

Comment:
It is possible to have a "more complete" test for the various enum values using [parametized tests](https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/)